core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -2369,6 +2366,97 @@ public static NsfwLevel of(final int value) {
         }
     }
 
+    /**
+     * Describes the features of a guild.
```

Comment:
I feel need mention the docs for https://discord.com/developers/docs/resources/guild#guild-object-guild-features
like the original method.